### PR TITLE
feat: add fingerprint concept page

### PR DESCRIPTION
**Pull request by nloding**
## Description

- REL-31 - add fingerprint concept page

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/concepts/what-are-fingerprints.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+<span class="base-alert info">
+  <span>
+    Need access to the data? Our [Reactors](https://developers.basistheory.com/concepts/what-are-reactors/) and [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) enable you to share, compute, and analyze anything with your sensitive data without it touching your systems.
+  </span>
+</span>
+
+## How It Works
+
+Every [Token Type](https://docs.basistheory.com/#token-types) created with the same data will have the same fingerprint. Fingerprints are:
```
Comment (koalaty-code) [inline collections/concepts/what-are-fingerprints.md:27]: Not all Token Types are fingerprinted. Should we update this to "Every fingerprinted Token Type..."? I don't think "fingerprintable" is a word, but I like that too.

File: collections/concepts/what-are-fingerprints.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What Is Fingerprinting?
+
+Tokenizing data replaces raw values with a Token Identifier referencing sensitive data, meaning you don’t have the raw value in your database. Without the raw value, your system isn’t able to perform simple queries that your application may require - For example, what if you need to check for duplicate or existing values? Fingerprints solve this problem by providing you an identifier representing your Token’s uniqueness without needing to read the decrypted data.
```
Comment (koalaty-code) [inline collections/concepts/what-are-fingerprints.md:17]:
```suggestion
Tokenizing data replaces raw values with a Token Identifier referencing sensitive data, meaning you don’t have the raw value in your database. Without the raw value, your system isn’t able to perform simple queries that your application may require - for example, what if you need to check for duplicate or existing values? Fingerprints solve this problem by providing you an identifier representing your Token’s uniqueness without needing to read the decrypted data.
```

File: collections/concepts/what-are-fingerprints.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    -H "BT-API-KEY: <BT_API_KEY>" \\
+    -X "POST"  \\
+    -D '{
+		"card": {
+          "number": "4242424242424242",
+          "expiration_month": 12,
+          "expiration_year": 2022,
+          "cvc": "123"
+        }
+	}'
```
Comment (koalaty-code) [inline collections/concepts/what-are-fingerprints.md:52]: Is the JSON indentation wrong here or is that just an issue in this view?

File: collections/concepts/what-are-fingerprints.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+For example, a social security number in a single tenant would generate the same fingerprint every time it is tokenized. For the same value in a different tenant, it would generate a different fingerprint, unique to that tenant.
+
+[Learn which token types support fingerprinting.](https://docs.basistheory.com/#token-types)
+
+Due to its non-sensitive nature, fingerprints can be stored in any environment. Even if the fingerprint were leaked or compromised there is no way to reverse it. Fingerprints are immune to brute force attacks because they are unique to your tenant.
+Take a look at this sample request and response:
+
+```js
+curl "<https://api.basistheory.com/atomic/cards>" \\
```
Comment (koalaty-code) [inline collections/concepts/what-are-fingerprints.md:41]: We are deprecating the `atomic/cards` endpoints. Should we switch this example to do a POST against `/tokens`? I believe the request/response model is identical except `card` -> `data`.

File: collections/concepts/what-are-fingerprints.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Common Use Cases
+
+### Fraud detection
+
+Financial systems need to detect and alert on possible fraud. Without viewing the credit card or bank information in plain text, it can be difficult to check whether you’ve seen that account before or not. Fingerprinting provides a simple answer to that problem. Imagine you receive two payments from the same credit card, one in New York City, the other in Los Angeles, just 10 minutes apart. That sounds fraudulent! Both tokens would have the same fingerprint, giving your system the ability to recognize the same card and take the appropriate action.
+
+### Identity protection
+
+The social security number token type supports fingerprinting and when  generated, a system can immediately detect whether that SSN is already associated with an existing account. All without the system searching through the underlying plaintext data or needing to decrypt or manage potentially reversible hashes itself.
```
Comment (koalaty-code) [inline collections/concepts/what-are-fingerprints.md:99]:
```suggestion
The social security number token type supports fingerprinting and when generated, a system can immediately detect whether that SSN is already associated with an existing account. All without the system searching through the underlying plaintext data or needing to decrypt or manage potentially reversible hashes itself.
```

File: collections/concepts/what-are-fingerprints.md (hunk 1 above)
Comment (nloding) [reply collections/concepts/what-are-fingerprints.md:27]: There is a "Learn which token types support fingerprinting" link below this - do you think that's still misleading?

File: collections/concepts/what-are-fingerprints.md (hunk 3 above)
Comment (nloding) [reply collections/concepts/what-are-fingerprints.md:52]: fixed

File: collections/concepts/what-are-fingerprints.md (hunk 5 above)
Comment (nloding) [reply collections/concepts/what-are-fingerprints.md:99]: fixed

File: collections/concepts/what-are-fingerprints.md (hunk 1 above)
Comment (koalaty-code) [reply collections/concepts/what-are-fingerprints.md:27]: Yeah, the latter contradicts this statement and we can't guarantee we capture the reader's attention throughout the entire document.

File: collections/concepts/what-are-fingerprints.md (hunk 4 above)
Comment (nloding) [reply collections/concepts/what-are-fingerprints.md]: makes sense - fixed

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**